launcher/src/test/java/org/opendaylight/etcd/launcher/test/EtcdLauncherTest.java

```diff
@@ -7,14 +7,15 @@
  */
 package org.opendaylight.etcd.launcher.test;
 
-import static com.google.common.base.Charsets.UTF_8;
 import static com.google.common.truth.Truth.assertThat;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
```

Comment:
This change seems to cause 2 new Checkstyle problems, would you like to fix that up?

```
[ERROR] src/test/java/org/opendaylight/etcd/launcher/test/EtcdLauncherTest.java:[14] (imports) CustomImportOrder: 'ch.vorburger.exec.ManagedProcessException' should be separated from previous import group by one line.
[ERROR] src/test/java/org/opendaylight/etcd/launcher/test/EtcdLauncherTest.java:[17] (imports) CustomImportOrder: Wrong lexicographical order for 'io.etcd.jetcd.ByteSequence' import. Should be before 'io.etcd.jetcd.KV'.
```

Reply:
There are actually a lot more problems like this in many files ... I've fixed them all up in bd0701a9bfdf519fcbac91ae035fc9846d662d58 (BTW FYI with https://github.com/vorburger/opendaylight-eclipse-setup you can get Eclipse automatically configured to do the right thing when you press Ctrl-Shift-O to Format Imports; I think something similar is possible for IntelliJ as well). With that, I get test failures in `odl-mdsal-broker` which we should fix...